frontend/app/src/components/ExternalTradeForm.vue

```diff
@@ -540,13 +540,17 @@ export default class ExternalTradeForm extends Vue {
       this.selectedCalculationInput = 'quoteAmount';
       this.$nextTick(() => {
         const quoteAmountInput = this.$refs.quoteAmountInput as any;
-        if (quoteAmountInput) quoteAmountInput.focus();
+        if (quoteAmountInput) {
+          quoteAmountInput.$el.querySelector('input').focus();
```

Comment:
Wouldn't it be a bit better to add a `focus` method to `AmountInput` and have it handle this? the $el is the v-text-field that is inside AmountInput, correct?

Reply:
Updated @kelsos, you mean like this?